core/src/main/java/org/ldaptive/auth/ext/FreeIPAAuthenticationResponseHandler.java

```diff
@@ -0,0 +1,105 @@
+/* See LICENSE for licensing and NOTICE for copyright. */
+package org.ldaptive.auth.ext;
+
+import java.util.Calendar;
+import org.ldaptive.LdapAttribute;
+import org.ldaptive.LdapEntry;
+import static org.ldaptive.ResultCode.*;
+import org.ldaptive.auth.AuthenticationResponse;
+import org.ldaptive.auth.AuthenticationResponseHandler;
+import org.ldaptive.io.GeneralizedTimeValueTranscoder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * Attempts to parse the authentication response and set the account state using data associated with eDirectory. The
+ * {@link org.ldaptive.auth.Authenticator} should be configured to return 'passwordExpirationTime' and
+ * 'loginGraceRemaining' attributes so they can be consumed by this handler.
+ *
+ * @author  tduehr
+ */
+public class FreeIPAAuthenticationResponseHandler implements AuthenticationResponseHandler
+{
+
+  /** Maximum password age. */
+  private int maxPasswordAge = -1;
+
+  /** Maximum password age. */
+  private int maxLoginFailures = -1;
```

Comment:
There's no way to set this variable. Looks like the variable is needed, I assume this is an oversight.